src/modules/extendedCallWindow/assets/vehicleListPermanentSearch.ts

```diff
@@ -0,0 +1,22 @@
+export default (): void => {
+    const vehicleTabs = document.querySelector('#tabs');
+    if (vehicleTabs) vehicleTabs.addEventListener('click', vehicleListChange);
+    //Initial expanding
+    vehicleListChange()
+
+    /**
+     * Gets called on every vehicle tab change to update the vehicle search bar
+     * (for some reasons they have different search bars instead of filters).
+     */
+    function vehicleListChange() {
+        const vehicleSearch = document
+            .querySelector('#vehicle_list_step>.tab-content>.active')
+            ?.querySelector(
+                "[id^='mission_vehicle_search_outer_']"
+            ) as HTMLInputElement;
```

Comment:
we could simplify this to a single querySelector:
```suggestion
            .querySelector<HTMLInputElement>('#vehicle_list_step>.tab-content>.active [id^="mission_vehicle_search_outer_"]');
```